assets/js/common_utils.js

```diff
@@ -96,38 +96,53 @@ export const getTime = () => {
   return `${hour}:${min}:${sec}`;
 };
 
-export const getOrtDevVersion = async () => {
-  const response = await fetch(
-    "https://cdn.jsdelivr.net/npm/onnxruntime-web/dist/"
-  );
-  const htmlString = await response.text();
-  const parser = new DOMParser();
-  const doc = parser.parseFromString(htmlString, "text/html");
-  let selectElement = doc.querySelector(".path li");
-  selectElement = doc.querySelector("select.versions.select-css");
-  let options = Array.from(selectElement.querySelectorAll("option")).map(
-    (option) => option.value
-  );
-  options = options.filter(option => !option.includes("esmtest"));
-  return options[0].replace("onnxruntime-web@", "");
+// Get the latest dev version of ONNX Runtime Web dists
+const getLatestOrtWebDevVersion = async () => {
+  try {
+    const response = await fetch('https://cdn.jsdelivr.net/npm/onnxruntime-web/dist/');
+    const htmlString = await response.text();
+    const parser = new DOMParser();
+    const doc = parser.parseFromString(htmlString, 'text/html');
+    let selectElement = doc.querySelector('select.versions.select-css');
+    let options = Array.from(selectElement.querySelectorAll('option')).map(
+      (option) => option.value
+    );
+    let filteredOptions = options.filter(item => item.includes('-dev.'));
+    return filteredOptions[0].replace('onnxruntime-web@', '');
+  } catch (error) {
+    console.error('Error:', error.message);
+  }
+};
+
+const ORT_BASE_URL = 'https://www.npmjs.com/package/onnxruntime-web/v/';
+const ORT_CDN_URL = 'https://cdn.jsdelivr.net/npm/onnxruntime-web@';
+const ortLink = (version) => `${ORT_BASE_URL}${version}?activeTab=versions`;
+
+const loadOrtScript = async (version, url) => {
+  try {
+    await loadScript('onnxruntime-web', url);
+    return `ONNX Runtime Web: <a href="${ortLink(version)}">${version}</a>`;
+  } catch (error) {
+    console.error('Failed to load ORT script:', error);
+    return 'Failed to load ONNX Runtime Web';
+  }
 };
 
 export const setupORT = async () => {
-  const ortversion = document.querySelector("#ortversion");
-  removeElement("onnxruntime-web");
-  let ortVersion = "1.18.0";
-  let ortLink = "";
-  if (ortVersion && ortVersion.length > 4) {
-    await loadScript(
-      "onnxruntime-web",
-      `https://cdn.jsdelivr.net/npm/onnxruntime-web@${ortVersion}/dist/ort.all.min.js`
-    );
-    ortLink = `https://www.npmjs.com/package/onnxruntime-web/v/${ortVersion}`;
-    ortversion.innerHTML = `ONNX Runtime Web: <a href="${ortLink}">${ortVersion}</a>`;
+  const ortVersionElement = document.querySelector('#ortversion');
+  removeElement('onnxruntime-web');
+  const queryOrt = getQueryValue('ort')?.toLowerCase();
+  let versionHtml;
+  if (queryOrt?.includes('-dev.')) {
+    versionHtml = await loadOrtScript(queryOrt, `${ORT_CDN_URL}${queryOrt}/dist/ort.all.min.js`);
+  } else if (queryOrt === 'test') {
+    await loadScript('onnxruntime-web', '../../assets/dist/ort.all.min.js');
```

Comment:
Why do the other 2 nearby lines call `loadOrtScript`, but this one calls `loadScript`? The subtle difference between them is unclear here, as they both take two parameters.

Reply:
@fdwr the two parameters are different for loadOrtScript and loadScript.

loadScript('onnxruntime-web', '../../assets/dist/ort.all.min.js') will generate 

`<script id="onnxruntime-web" src="../../assets/dist/ort.all.min.js"></script>`

while the loadOrtScript not only calls the `await loadScript('onnxruntime-web', url);` but also handles some messages inluding `ONNX Runtime Web: <a href="${ortLink(version)}">${version}</a>`

Reply:
@fdwr Renamed the function name, PTAL

Reply:
Yeah, new name is clearer (before they both loaded the ORT script :b).